src/Permissions/Widgets/AppSettingsView.vala

```diff
@@ -113,16 +129,61 @@ public class Permissions.Widgets.AppSettingsView : Switchboard.SettingsPage {
                 should_enable_reset = true;
             }
 
-            list_box.append (override_row);
+            sandbox_box.append (override_row);
+        });
+
+        var permission_store = PermissionStore.get_default ();
+
+        background_row = new PermissionSettingsWidget (
+            _("Background Activity"),
+            _("Perform tasks and use system resources while its window is closed."),
+            "permissions-background"
+        );
+
+        background_row.notify["active"].connect (() => {
+            string[] permissions = { };
+            if (background_row.active) {
+                permissions += "yes";
+            } else {
+                permissions += "no";
+            }
```

Comment:
These `+=`'s make it hard to understand that `permissions` always contains only one item. Maybe replace it with something like:

```
string[] permissions;
if (background_row.active) {
    permissions = { "yes" };
} else {
    permissions = { "no" };
}
```

or even

```
string[] permissions = { background_row.active ? "yes" : "no" };
```

Reply:
That's way cleaner and clearer, thanks for the suggestion!